api/client/batch_client.py

```diff
@@ -10,7 +11,7 @@
 
 from tornado import gen
 from tornado.escape import json_decode
-from tornado.httpclient import AsyncHTTPClient, HTTPRequest, HTTPError
+from tornado.httpclient import AsyncHTTPClient, HTTPRequest, HTTPClientError
```

Comment:
https://www.tornadoweb.org/en/stable/httpclient.html#tornado.httpclient.HTTPClientError

The name of this exception was changed in Tornado, so I updated it here as well and added the minimum version to the requirements.txt where the name change took place.